package.json

```diff
@@ -39,15 +39,15 @@
     "husky": "^4.2.3",
     "jest": "^24.9.0",
     "prettier": "^1.18.2",
-    "react": "^16.9.0",
-    "react-native": "0.60.5",
+    "react": "^16.13.0",
+    "react-native": "0.63.0",
```

Comment:
Expo currently uses 0.62. Should try to stick to that as the target since a huge portion of the community uses expo

Reply:
Sure fix `react-native` to 0.62.2 and `react` to 16.11.0